src/main/java/se/kth/dd2480/grp25/ci/CloneJob.java

```diff
@@ -36,12 +36,9 @@ public CloneJob(Event event) {
    * @return an print job represented by a {@link Runnable} if accepted
    */
```

Comment:
Just noticed this now, the return should be a clone job not a print job :)

Reply:
Good that you spotted this :) fixed!